_posts/en/newsletters/2020-12-16-newsletter.md

```diff
@@ -0,0 +1,331 @@
+---
+title: 'Bitcoin Optech Newsletter #128'
+permalink: /en/newsletters/2020/12/16/
+name: 2020-12-16-newsletter
+slug: 2020-12-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Proposed improvements to static LN backups:** anyone receiving money
+  in a payment channel, including those currently used by LN, needs to
+  keep track of the channel's latest state---all the details that make
+  it possible to close the channel and receive the correct share of the
+  channel's funds onchain.  Unfortunately, computers have a bad habit of
+  losing data and regular periodic backups don't help much when a
+  channel's state could've changed just milliseconds before a disk drive
```

Comment:
I see you're an unfortunate victim of the teaching that contractions shouldn't be used in formal writing.  I'm from the school that thinks they're a wonderful tool for making sentences shorter, easier to read, and more natural sounding.  (Of course, we ain't wanna be abusin' 'em.)

Reply:
I usually don't mind contractions, but in the case of "could have changed", "could've changed" reads more difficult to me. :shrug:

Reply:
Interesting.  Do you have a feeling for whether that's something specific to you or whether it's something that might affect other readers of English as a second language?

Reply:
I don't think I'd ever say "could've" (I'm not even sure how to pronounce that without slipping at least an "a" in there), but would always use that uncontracted as in "could have". Maybe that's what's throwing my reading off. Can't speak for the general population, but it may just be that the contraction appears seldom in written English.

Reply:
My experience is that the Europeans I know were taught the same as @Xekyo, and I was as well (in the US by a hardcore, demanding but amazing English teacher). Maybe it's old school compared to the current state in the US, I don't know.